substrate/primitives/consensus/common/src/error.rs

```diff
@@ -56,4 +56,13 @@ pub enum Error {
 	/// Some other error.
 	#[error(transparent)]
 	Other(#[from] Box<dyn std::error::Error + Sync + Send + 'static>),
+	/// Some other error with string message.
+	#[error("Other error: {0}")]
+	OtherString(String),
+}
+
+impl From<String> for Error {
+	fn from(s: String) -> Self {
+		Error::OtherString(s)
+	}
```

Comment:
```suggestion
}
```

`Box::new(String::new("Whatever"))` implement `Error + Sync + Send`: https://doc.rust-lang.org/std/boxed/struct.Box.html#impl-From%3C%26str%3E-for-Box%3Cdyn+Error+%2B+Send+%2B+Sync%3E